### ISSUE TITLE
CI Sandbox for Source WooCommerce

### ISSUE THREAD

**Issue by bazarnov** [airbytehq/airbyte#5859]
## Tell us about the new connector you’d like to have
We need to merge the contribution of https://github.com/airbytehq/airbyte/pull/5121 (source WooCommerce).
For this to be properly tested before merge, and afterwards using SAT, we would need to have the Sandbox for WooCommerce.

As for Sandbox for WooCommerce, there is an official reply from the WooCommerce:
```
> We’ve received requests to build an integration with the Woocommerce API in order to allow our mutual users to perform
> analytics on their Woocommerce data. We’re happy to build the integration ourselves, but would need access to a sandbox
> account that we can use to test our integration against.

I see you've started working on that already at https://airbyte.io/connectors/woocommerce. That's fantastic!
WooCommerce itself is open source, so you are welcome to download it and run your testing! Stores that use WooCommerce run from the owner's hosting server rather than from a central server managed by us. You can download WordPress and install it on most hosts, then add WooCommerce to test it out. Since the plugin is free, we don't manage a sandbox for others to test out.
You can download WordPress here: https://wordpress.org/download/
This account would be used solely for testing purposes and we would not be using it in place of purchasing a paid plan in case we were to properly use the product as a normal customer would.
It looks like you've started making a site on WordPress.com. A Business plan would be needed to use our managed hosting to create your test site and install plugins, as WordPress.com does not currently offer development plans. A plan is not needed if you opt to run your test site on another host. You might create a local site, or install WordPress to a site on your current server to test it out.
You can read about the difference here: https://en.support.wordpress.com/com-vs-org/
We have a wealth of developer resources online here to check out: https://developer.woocommerce.com/
All about the REST API is here: https://woocommerce.github.io/woocommerce-rest-api-docs/
You can also visit the #developers channel of our Community Slack if you have questions.
```

## Describe the context around this new connector
In order to have the Sandbox we need to have the hosted resource for that, or build the site using free wordpress plugin available for free, hosted on our side, enable the paid plugins, populate with the data and perform the tests on it on the regular basis.

## Describe the alternative you are considering or using
Buy the paid version directly from WooCommerce and host the account on their site, indefinitely, if possible.

**Comment by igrankova** [comment]
Decided to postpone host registering for the WooCommerce testing:
To test WooCommerce we need to host on the Wordpress (or any other hosting).
The WooCommerce representative sent us the next info: 
WordPress.com does offer hosting, if you choose to use that to test out WooCommerce, you'll need the Business plan.
Costs of the Business Plan: $33 per month, billed monthly or $25 per month, billed yearly ($300). 
All plans pricing is here. https://wordpress.com/pricing/        
Monthly pricing is more expensive and there's no free domain for one year included comparing to the yearly payment plan.

**Comment by YowanR** [comment]
@igrankova @ycherniaiev Can you help get us this account back up? Let's go for the monthly plan so we are unblocked.

**Comment by igrankova** [comment]
@YowanR, the plan price raized up to 40$ monthly:

**Comment by igrankova** [comment]
@YowanR, @bazarnov  The plan was upgraded to Business

**Comment by YowanR** [comment]
This should unblock us now. cc @lazebnyi @YuliiaNahoha -- can we see if WooCommerce can be migrated over now?

**Comment by lazebnyi** [comment]
So, if we have sandbox. We can start working on migration.
